gcloud-sdk/src/token_source/ext_creds_source/mod.rs

```diff
@@ -42,11 +63,26 @@ pub async fn subject_token(
     client: &reqwest::Client,
     external_account: &ExternalAccount,
 ) -> crate::error::Result<SecretValue> {
-    match external_account.credential_source {
+    match &external_account.credential_source {
         ExternalCredentialSource::UrlBased(ref url_creds) => {
             subject_token_url(client, url_creds).await
         }
         ExternalCredentialSource::FileBased(ref url_creds) => subject_token_file(url_creds).await,
+        #[cfg(feature = "external-account-aws")]
+        ExternalCredentialSource::Aws(Aws {
+            regional_cred_verification_url,
+            ..
```

Comment:
For simplicity, current implementation does not fallback to `region_url` if region cannot be determined from env or imds.